architecture/core_concepts/routes.adoc

```diff
@@ -12,22 +12,31 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
+An OpenShift link:../../architecture/core_concepts/routes.html[route] exposes a
```

Comment:
We're starting to use `xref` instead of `link`.  So this needs to change to:

```asciidoc
xref:../../architecture/core_concepts/routes.adoc#[route]
```

Note that in addition to `s/link:/xref:/`, there is also `s/html/.adoc#/` (note trailing `#`).  I will omit commenting on the other instances in the file, but generally they should undergo the same transform.